specs/data_structures.md

```diff
@@ -346,18 +343,13 @@ For some intuition: the min and max namespace IDs for subtree roots with at leas
 
 A compact commitment can be computed by taking the [hash](#hashing) of the [serialized](#serialization) root node.
 
-#### NamespaceMerkleTreeProof
+#### NamespaceMerkleTreeInclusionProof
 
-| name                 | type                             | description                   |
-| -------------------- | -------------------------------- | ----------------------------- |
-| `rootHash`           | [HashDigest](#hashdigest)        | Root hash.                    |
-| `rootNamespaceIDMin` | [NamespaceID](#type-aliases)     | Root minimum namespace ID.    |
-| `rootNamespaceIDMax` | [NamespaceID](#type-aliases)     | Root maximum namespace ID.    |
-| `key`                | `byte[32]`                       | Key (i.e. index) of the leaf. |
-| `siblingValues`      | [HashDigest](#hashdigest)`[]`    | Sibling hash values.          |
-| `siblingMins`        | [NamespaceID](#type-aliases)`[]` | Sibling min namespace IDs.    |
-| `siblingMaxes`       | [NamespaceID](#type-aliases)`[]` | Sibling max namespace IDs.    |
-| `leaf`               | `byte[]`                         | Leaf value.                   |
+| name            | type                             | description                                                     |
+| --------------- | -------------------------------- | --------------------------------------------------------------- |
+| `siblingValues` | [HashDigest](#hashdigest)`[]`    | Sibling hash values, ordered starting from the leaf's neighbor. |
+| `siblingMins`   | [NamespaceID](#type-aliases)`[]` | Sibling min namespace IDs.                                      |
+| `siblingMaxes`  | [NamespaceID](#type-aliases)`[]` | Sibling max namespace IDs.                                      |
```

Comment:
`NamespaceMerkleTreeInclusionProof` is the inclusion proof for a single node right? Isn't `siblingMins` and `Maxes` implicitly included in `siblingValues`?

Reply:
Yes, it's only for the inclusion of a single leaf. The min and max sibling values are committed to in the hashes, but they're not known to a verifier. So they still need to be provided.